pkg/blob/nodeserver.go

```diff
@@ -317,7 +317,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		source := fmt.Sprintf("%s:/%s/%s", serverAddress, accountName, containerName)
 		mountOptions := util.JoinMountOptions(mountFlags, []string{"sec=sys,vers=3,nolock"})
 		if err := wait.PollImmediate(1*time.Second, 2*time.Minute, func() (bool, error) {
-			return true, d.mounter.MountSensitive(source, targetPath, NFS, mountOptions, []string{})
+			return true, d.mounter.MountSensitive(source, targetPath, AZNFS, mountOptions, []string{})
```

Comment:
pls also add a driver feature flag to control this behavior, by default, we are using aznfs mount, and if feature flag is set as false, then we use default nfs mount, that would be safer.

Reply:
Good suggestion